filebeat/docs/configuration.asciidoc

```diff
@@ -148,6 +148,50 @@ document. The default value is `log`.
 
 The buffer size every harvester uses when fetching the file. The default is 16384.
 
+===== max_bytes
+
+Maximum number of bytes a single log event can have. All bytes after max_bytes are discarded and not sent.
+This is especially useful for multiline log messages which can get large. The default is 10MB (10485760).
+
+===== multiline
+
+Mutiline can be used for log messages spanning multiple lines. This is common for Java Stack Traces
+or C-Line Continuation. The following example combines all lines following a line start with a `[`.
+This could for example be a Java Stack Trace.
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+multiline:
+    pattern: ^\[
+    match: after
+    negate: true
+-------------------------------------------------------------------------------------
+
+====== pattern
+
+The regexp Pattern that hast to be matched. The example pattern matches all lines starting with [
```

Comment:
we're using [CompilePOSIX](https://golang.org/pkg/regexp/#CompilePOSIX) to compile the regular expressions. These are 'mostly' compatible to egrep + some quirks (see documentation).

TBH I find it quit difficult to understand the impact on syntax when given this line in documentation:

>  restricts the regular expression to POSIX ERE (egrep) syntax